grails-app/controllers/org/broadinstitute/orsp/IssueListController.groovy

```diff
@@ -19,10 +19,15 @@ class IssueListController extends AuthenticatedController {
         if (isORSP()) {
             users.add(SupplementalRole.ORSP)
         }
+
         if (isComplianceOffice()) {
             users.add(SupplementalRole.COMPLIANCE_OFFICE)
         }
 
+        if (isReadOnlyAdmin()) {
+            users.add(SupplementalRole.RONLY_ADMIN)
```

Comment:
Nit: naming, I would prefer that this be spelled out, `RONLY` looks a bit like `RONY` which is short for pepperoni pizza in these parts 😄 
```suggestion
            users.add(SupplementalRole.READ_ONLY_ADMIN)
```